src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -85,6 +86,7 @@ export default class UIOptionsComponent extends React.Component {
 			<div className="col-xs-2" style={divPadding}>
 				<div className="col-xs-11">
 					<div>
+						<MetersFilterContainer />
```

Comment:
I think this entire search thing should be one component with a search bar and a dropdown list (both components themselves), instead of putting both as siblings `UIOptionsComponent`. You also aren't actually using the search term anywhere yet.